core/templates/dev/head/exploration_editor/main_tab/StateResponses.js

```diff
@@ -284,20 +286,26 @@ oppia.controller('StateResponses', [
       trainingDataService.initializeTrainingData(
         explorationId, currentStateName);
     };
+
     $scope.suppressDefaultAnswerGroupWarnings = function() {
       var interactionId = $scope.getCurrentInteractionId();
+      var answerGroups = responsesService.getAnswerGroups();
+      var handledAnswersArray = [];
+      var choiceIndices = [];
+      var answerChoices = [];
+      var choiceRules = [];
+      var customizationArgs = stateCustomizationArgsService.savedMemento;
```

Comment:
Hi @anthkris, I tried your branch locally and in console I got bunch of `Cannot read property 'value' of undefined`. Also I have tip to solve that error.

First of all `stateCustomizationArgsService` has init function. And from your code I am positive that you did not initialise it using that function.
 so first do something like below 
```
var stateName = editorContextService.getActiveStateName();
var stateData = explorationStatesService.getState(stateName);
stateCustomizationArgsService.init(stateName, stateData.interaction.customization_args);
```  

This is to be done before you make a call to service itself (before this line-297) since `savedMemento` return the value you pass to service in `init()` function. 

My suggestion is not a solution. I mean using above hack I was able to get rid of console warnings but I am not sure that values which I passed to `init()` function are correct for this fix. It just meant to give you an idea where you are going wrong. Also check the broadcast event `stateEditorInitialized` in `StateEditor.js` and watcher set on this event in `StateInteraction.js` for more information.

I hope this helps.